doc/source/conf.py

```diff
@@ -22,13 +22,19 @@
 import os
 import pathlib
 import re
+from datetime import datetime
 
 # -- Project information -----------------------------------------------------
 # https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information
 
-project = 'Apache Arrow Flight SQL adapter for PostgreSQL'
-copyright = '2022-2023, Apache Arrow Developers'
-author = 'Apache Arrow Developers'
+project = 'Apache Arrow Flight SQL Adapter for PostgreSQL'
+copyright = f'2019-{datetime.now().year}, Apache Software Foundation'
+trademark = [
+    "Apache Arrow Flight, Arrow Flight, Apache, the Apache feather logo, and the Apache Arrow project logo",
```

Comment:
Should we add some words for "Flight SQL" here too?

Hmm, ADBC doesn't mention Flight nor Flight SQL: https://github.com/apache/arrow-adbc/pull/1423

Reply:
Yes sounds good, added a proposal by explicitly mentioning Arrow Flight SQL